PSDesiredStateConfiguration.md

```diff
@@ -0,0 +1,106 @@
+## Getting Started
+
+**VMware.PSDesiredStateConfiguration** provides a new way to compile and execute **DSC Configurations** on **Windows**, **Linux** and **MacOS** that does not require the use of **DSC LCM** or the typical MOF format and without altering the existing **Configuration** syntax.  Instead the cmdlets of this module create and work with regular powershell objects. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.  
+
+## Requirements
+
+The following table describes the required dependencies for running VMware.vSphereDSC Resources.
+
+ **Required dependency**   | **Minimum version**
+-------------------------- | -------------------
+`PowerShell`               | 5.1 (Core is supported)
+
+1. Copy the VMware.PSDesiredStateConfiguration Module to one of the system PowerShell module directories. For more information on installing PowerShell Modules, please visit [Installing a PowerShell Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-7).
```

Comment:
This should should go in separate paragraph "Install". Install guide should have subsections for manual install from code (when user takes it from github) and install from PSGallery (this would be added once we have it published)

Change the text to "Make sure VMware.PSDesiredStateConfiguration Module is available in PowerShell.

### Install manually

Copy the module folder into one of the PSModulePath directories. 
### Install from PSGallery 
Use `Install-Module  VMware.PSDesiredStateConfiguration`. For more information on installing PowerShell Modules, please visit [Installing a PowerShell Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-7)"